source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,62 @@
+
+.. image:: images/robot-battery/heatshrink.png
+   :alt: One terminal of an FRC battery fully covered in heatshrink.
+
+It is a good idea to secure the battery with either a star washer or a nord lock washer as this will ensure the connection does not move.  This connection must also be wrapped, while some use electrical tape heatshrink that fits over the entire connection is recommended.
+
+Battery Longevity
+^^^^^^^^^^^^^^^^^
+
+This battery is rated for about 1200 charge/recharge cycles, but for the high currents FRC subjects it to that lifespan is reduced to about 400 cycles.  These cycles are intended to be relatively limited discharge and deep cycling the battery (running it all the way down) can damage it.
+
+Batteries last the longest if they are kept fully charged when not it use.  Take care however to not leave them on a trickle charger for an extended period of time.
+
+Dropping a battery may compromise its structural integrity and can lead to a bulge or crack in the battery case.  These batteries should be safely disposed of as soon as possible.
+
+Battery Load Tester
+^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/robot-battery/load-tester.jpg
+   :alt: Output screen of a common load tester showing properties of the battery.
+
+A battery load tester can be used as a quick way to determine the readiness of a battery.  It may provide information like: open-load voltage, voltage under load, internal resistance, and state of charge.  These metrics can be used to quickly confirm that a battery is ready for a match and even possibly help to identify some long term problems with the battery.
+
+While these batteries are called 12 volt batteries a fully charged battery should read around 13 volts.  Ideal internal resistance should be around 0.015 Ohms.  If a battery gets higher than 0.02 Ohms it is a good idea to consider not using that battery for competition matches.
+
+Battery Analyzer
+^^^^^^^^^^^^^^^^
+
+.. image:: images/robot-battery/cba-graph.jpg
+   :alt: Graph from a common battery analyzer plotting volts and AmpHrs for many different batteries.
+
+A battery analyzer is used to give a detailed inspection of battery performance.  It will provide graphs of battery performance over time.  This test takes significant time (roughly two hours) so it is less suited to testing during competition.  It is recommended to run this test on each battery every year to monitor and track its performance.  This will determine how it should be used: matches, practice, testing, or disposed of.
+
+Battery Best Practices
+^^^^^^^^^^^^^^^^^^^^^^
+
+Teams should purchase several new batteries each year to help keep their batteries fresh.  Elimination matches can require many batteries and because of the quick turn around time batteries may not have enough time to recharge.
```

Comment:
Perhaps use bullets in this section to emphasize that each portion is "standalone".